dear-imgui.cabal

```diff
@@ -56,6 +56,8 @@ library
     if os(windows)
       extra-libraries:
         opengl32
+    if os(darwin)
+      frameworks: OpenGL
```

Comment:
Do you mind submitting this in a separate PR? It's not quite related to haskell.nix stuff, and I like to keep PRs specific.